plugins/generic/pln/classes/DepositPackage.inc.php

```diff
@@ -192,9 +192,36 @@ function generateAtomDocument() {
 				$pkpDetails->setAttribute('checksumValue', md5_file($packageFile));
 				break;
 		}
-
+		
 		$entry->appendChild($pkpDetails);
 		$atom->appendChild($entry);
+		
+		$license = $atom->createElementNS('http://pkp.sfu.ca/SWORD', 'license');
+		$license->appendChild($atom->createElementNS('http://pkp.sfu.ca/SWORD', 'openAccessPolicy', $journal->getLocalizedSetting('openAccessPolicy')));
```

Comment:
This will run from an acron job, so it will pick up the user's locale. I don't think I quite understand OJS's locale system.

I've used getSetting() for things that don't seem translatable/localizable, eg. licenseURL, copyrightYearBasis, and copyrightHolderType. Should I have used the localized setting instead?